lib/config/streams.js

```diff
@@ -51,11 +30,11 @@ function jsApp(opts) {
 }
 
 function jsLib(opts) {
-  return gulp.src(getLocalLibGlob()('**/*.js', '!*.js', '!**/*.spec.js'), opts);
+  return gulp.src(getLocalLibGlob()('**/*.js', '!**/*.spec.js'), opts);
 }
 
 function jsSpec(opts) {
-  return gulp.src(getLocalLibGlob()('**/*.spec.js', '!*.spec.js'), opts);
```

Comment:
project directory is no longer included so we don't need to ignore its `spec.js` files